templates/crate/details.html

```diff
@@ -31,7 +31,7 @@
                         {# If the crate has a homepage, show it #}
                         {%- if details.homepage_url -%}
                             <li class="pure-menu-item">
-                                <a href="{{ details.homepage_url }}" class="pure-menu-link">
+                                <a href="{{ details.homepage_url | safe }}" class="pure-menu-link">
```

Comment:
This is user-controlled input. I am not comfortable putting a blanket `| safe` here, no one should be trying to parse this anyway.